cypress/integration/form_spec.js

```diff
@@ -1,19 +1,24 @@
 describe ('Form user flow', () => {
   beforeEach(() => {
     cy.visit('/')
+      .intercept("https://api.nytimes.com/svc/topstories/v2/home.json?api-key=GKUzDD1VY9ssjZ1AGusX3ci6AeoXCaSr", {fixture: 'nyt-article-fetch.json'})
+      .intercept('https://api.dandelion.eu/datatxt/sent/v1/?lang=en&text=A new biography by Debby Applegate recounts the story of Polly Adler, who arrived in America from Russia at 13 and became New Yorks most successful brothel owner, befriending mobsters, policemen, politicians and writers&token=beb0091844524790b7672a69bac06a2a', {fixture: 'manhattan-madam-sentiment.json'})
+      .intercept('https://api.dandelion.eu/datatxt/sent/v1/?lang=en&text=The magazines Ethicist columnist on responding to infidelity&token=beb0091844524790b7672a69bac06a2a', {fixture: 'affair-sentiment.json'})
   })
 
   it('should render the appropriate elements', () => {
     cy.contains('WellNews')
-      .get('.cy-sentiment-question').contains('how are you feeling today?')
-      .get('.cy-happy-btn').should('be.visible')
+      .get('.cy-sentiment-question').contains('What kind of mood are you in?')
+      .get('.cy-completely-positive-btn').should('be.visible')
+      .get('.cy-positive-btn').should('be.visible')
       .get('.cy-neutral-btn').should('be.visible')
-      .get('.cy-sad-btn').should('be.visible')
+      .get('.cy-negative-btn').should('be.visible')
+      .get('.cy-completely-negative-btn').should('be.visible')
       .get('.cy-view-synopsis-button').should('be.visible')
   })
 
-  it('should take you to the news feed when the happy sentiment button is chosen', () => {
-    cy.get('.cy-happy-btn').click()
+  it('should take you to the news feed when the completely positive sentiment button is chosen', () => {
+    cy.get('.cy-completely-positive-btn').click()
       .url().should('include', '/feed')
   })
```

Comment:
Should we add tests for all of the buttons, since we're already doing these handful?